pos_access_right/static/src/xml/NumpadWidget.xml

```diff
@@ -16,6 +16,19 @@
             >{'disabled-mode': !hasMinusControlRights}</attribute>
             <attribute name="t-att-disabled">!hasMinusControlRights</attribute>
         </xpath>
+
+        <xpath
+            expr="div[hasclass('numpad')]//button[hasclass('numpad-backspace')]"
+            position="attributes"
+        >
+            <attribute
+                name="t-att-class"
+            >{'disabled-mode': !hasDeleteItemControlRights}</attribute>
+            <attribute name="t-att-disabled">!hasDeleteItemControlRights</attribute>
```

Comment:
```suggestion
            <attribute
                name="t-att-class"
                t-translation="off"
            >{'disabled-mode': !hasDeleteItemControlRights}</attribute>
            <attribute name="t-att-disabled" t-translation="off">!hasDeleteItemControlRights</attribute>
```

Could you add `t-translation="off"` to the attributes, to prevent odoo from trying to translate this string/value?
Otherwise.. bugs can arise from translations, which isn't good.

(please correct the other attributes as well.. 🙏🏻 )

Reply:
sorry for delay response. Could you please review again. Thanks.